modules/read_data.rb

```diff
@@ -0,0 +1,49 @@
+require 'json'
+require 'pry'
+
```

Comment:
- [ ] There is an error raised here when I try to run the app, Please fix that.

![image](https://github.com/shafiuyushawu/school-liabrary/assets/68796232/1c5d47d2-d573-4c47-9911-133ba73af57e)

as suggested in the terminal log above using `pty` instead of `pry` fixes the problem.